frontend/packages/dev-console/src/components/monitoring/__tests__/MonitoringPage.spec.tsx

```diff
@@ -52,11 +54,12 @@ describe('Monitoring Page ', () => {
     const actualTabs = component
       .find(utils.HorizontalNav)
       .prop('pages')
-      .map((page) => page.name);
+      .map((page) => t(page.nameKey));
```

Comment:
Maybe you can just remove the namespace from the i18n key?

```suggestion
      .map((page) => page.nameKey.replace('devconsole~', ''));
```